portalnet/src/overlay/service.rs

```diff
@@ -948,7 +948,7 @@ where
                     // Generate a connection ID for the uTP connection.
                     let enr = self.find_enr(source).ok_or_else(|| {
                         OverlayRequestError::AcceptError(
-                            "unable to find ENR for NodeId".to_string(),
+                            "handle_offer_content: unable to find ENR for NodeId".to_string(),
```

Comment:
should this be `handle_find_content`? that's the function we're inside at this point